Trappist/src/Promact.Trappist.Repository/TestConduct/TestConductRepository.cs

```diff
@@ -295,7 +295,7 @@ public TestConductRepository(TrappistDbContext dbContext, ITestsRepository testR
                 code.Input = testCase.TestCaseInput;
                 var result = await ExecuteCodeAsync(code);
 
-                if(result.ExitCode == -34494 || result.CompilerOutput != "")//EXITCODE if code run fails
+                if (result.ExitCode == -34494 || result.CompilerOutput != "")//EXITCODE if code run fails
```

Comment:
I did not write this line,  I worked in this repository but I did not make any changes like this